integrations/cloudbuild/build-all.yaml

```diff
@@ -0,0 +1,36 @@
+# NOTE: /workspace/ is the persistent directory across steps for
+steps:
+    - name: "connectedhomeip/chip-build-vscode:latest"
+      id: "CompileAll"
+      entrypoint: "./scripts/run_in_build_env.sh"
+      args:
+          [
+              "./scripts/build/build_examples.py --platform all build
+              --copy-artifacts-to /workspace/artifacts/",
+          ]
+      timeout: 7200s
+
+    # Final steps: upload artifacts. Use this because the artifacts/objects command does not seem to
+    # support recursive copies.
+    #
+    # This has the unfortunate sideffect that 'artifacts' will not be visible as part of the
+    # build run information.
```

Comment:
Can you mark a TODO here please? I don't want to be stuck with this versus archives and the builtin support.